### PR TITLE
make RBDVolumeSource consistent with comments

### PR DESCRIPTION
**Pull request by rootfs**
@kubernetes/rh-storage

### CODE REVIEW

**Comment by markturansky** [comment]
@bgrant0607 for API change.

File: pkg/api/v1/types.go [hunk 1]
```diff
@@ -545,15 +545,15 @@ type RBDVolumeSource struct {
 	// The rados pool name.
 	// Default is rbd.
```
Comment (bgrant0607) [inline pkg/api/v1/types.go]: These comments are incorrect. "rbd" is not a default value AFAICT.

Default values are set explicitly:
https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api-conventions.md#defaulting

If they were set, it should be here:
https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/defaults.go

We can't change this in v1 without breaking backward compatibility. Please file an issue to fix this.

The terminology I suggested elsewhere is "Implicitly inferred to be rbd if unset."

**Comment by thockin** [comment]
@rootfs - ping.  update or close?

**Comment by rootfs** [comment]
Will open a new one addressing the feedback

**Pull request closed without merging**